cns/dockerclient/dockerclient.go

```diff
@@ -150,12 +135,9 @@ func (dockerClient *DockerClient) CreateNetwork(networkName string, options map[
 	}
 
 	if enableSnat {
-		cmd := fmt.Sprintf("iptables -t nat -A POSTROUTING -m iprange ! --dst-range 168.63.129.16 -m addrtype ! --dst-type local ! -d %v -j MASQUERADE",
-			primaryNic.Subnet)
-		err = executeShellCommand(cmd)
+		err = common.SetOutboundSNAT(nicInfo.Subnet)
```

Comment:
For iptables commands, we can use this library:
https://github.com/coreos/go-iptables